# 删除UI

Header header

# 命令
# 删除图层
uint8 CMD_LAYER = 1
# 删除所有
uint8 CMD_ALL = 2
uint8 cmd

# 图层数,范围0-9
uint8 layer